### PR TITLE
[WIP] Revert k8s tests

### PR DESCRIPTION
**Pull request by galal-hussein**

### CODE REVIEW

**Comment by bgehman** [comment]
@galal-hussein is there documentation or special prereqs need to build rke with these tests?  I can build rke locally out of master (after the tests were removed in #143 ).  Building with this branch and it fails:

```
...
...
Successfully tagged rke:revert-k8s-test
Sending build context to Docker daemon  26.64MB
Step 1/2 : FROM rke:revert-k8s-test
 ---> cffe461d2549
Step 2/2 : COPY . /go/src/github.com/rancher/rke/
 ---> fc2884ed7df1
Removing intermediate container f2f17ee206fc
Successfully built fc2884ed7df1
Successfully tagged rke:revert-k8s-test
Running tests
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
./test: line 7: kill: (2330) - No such process
FATA[0046] exit status 1                                
make: *** [ci] Error 1
```

Apologize if this is the wrong place to ask...

**Comment by galal-hussein** [comment]
@bgehman we are testing using Rancher Dapper, you must have docker installed locally and run `make test` which will create the test environment via dapper and run the tests properly

File: scripts/test [hunk 1]
```diff
@@ -1,10 +1,92 @@
 #!/bin/bash
 set -e
 
+cleanup()
+{
+    local exit=$?
+    kill $PID || true
```
Comment (bgehman) [inline scripts/test]: mind adding this just before the `kill`?:

```
    if [ $exit -ne 0 ]; then
      cat /tmp/docker.log
    fi
```

using it to help figure out why I can't build locally.  After adding it, I'm seeing:

```
Removing intermediate container 45dfe0adb8b3
Successfully built 33a9550693c9
Successfully tagged rke:revert-k8s-test
Running tests
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
time="2017-12-14T16:49:07.853733195Z" level=debug msg="Warning: could not change group /var/run/dind.sock to docker: Group docker not found" 
time="2017-12-14T16:49:07.853998482Z" level=debug msg="Server created for HTTP on unix (/var/run/dind.sock)" 
time="2017-12-14T16:49:07.854498804Z" level=debug msg="Using default logging driver json-file" 
time="2017-12-14T16:49:07.854615179Z" level=debug msg="[graphdriver] trying provided driver \"aufs\"" 
time="2017-12-14T16:49:07.859712025Z" level=debug msg="Cleaning up old shm/mqueue mounts: start." 
time="2017-12-14T16:49:07.859872770Z" level=fatal msg="Error starting daemon: error initializing graphdriver: driver not supported" 
./scripts/test: line 10: kill: (12) - No such process
FATA[0024] exit status 1                                
make: *** [test] Error 1
```

Many thanks!

**Comment by alena1108** [comment]
Closing for now

**Pull request closed without merging**